### PR TITLE
Update cpu-z.json

### PR DESCRIPTION
**Pull request by RavenMacDaddy**
- Updates json to v1.98

### CODE REVIEW

**Comment by rashil2000** [comment]
`checkver` is missing

**Comment by RavenMacDaddy** [comment]
Sadly it's always been missing.

Personally I don't understand how regex works.

Hopefully someone has expertise on that and can improve the manifest.

File: bucket/cpu-z.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "version": "1.96.1",
+    "version": "1.98",
     "description": "System information software",
     "homepage": "https://www.cpuid.com/softwares/cpu-z.html",
     "license": "Freeware",
-    "url": "http://download.cpuid.com/cpu-z/cpu-z_1.96-en.zip",
-    "hash": "c8445a136b2a63a356e558dfc6e500d625859f05afe36e75391e94f407382f61",
+    "url": "http://download.cpuid.com/cpu-z/cpu-z_1.98-en.zip",
```
Comment (pratikpc) [inline bucket/cpu-z.json:6]: Can we switch the URLs to https as well?

Both here and in autoupdate?

**Comment by rashil2000** [comment]
You can specify `"checkver": "Version ([\\d.]+) for windows"`

**Comment by pratikpc** [comment]
@rashil2000 I will create a PR for the same

Just checkver present.

Based on offline tests, it works

File: bucket/cpu-z.json [hunk 2]
```diff
@@ -36,7 +36,8 @@
         }
     },
     "persist": "cpuz.ini",
+    "checkver": "Version ([\\d.]+) for windows",
     "autoupdate": {
-        "url": "http://download.cpuid.com/cpu-z/cpu-z_$version-en.zip"
+        "url": "https://download.cpuid.com/cpu-z/cpu-z_$version-en.zip"
     }
 }
```
Comment (pratikpc) [inline bucket/cpu-z.json:43]: I believe we need a new line at the end of file

Comment (RavenMacDaddy) [reply bucket/cpu-z.json:43]: There aren't any new brackets or errors on preview, shouldn't be needed normally in that case - or do you just mean an empty line?

Comment (pratikpc) [reply bucket/cpu-z.json:43]: Empty line

checkver.ps1 adds one to the end of file.

But I noticed a lot of files don't have them.

Let me check with /verify once

**Review by rashil2000 — APPROVED** [review]

File: bucket/cpu-z.json (hunk 2 above)
Comment (pratikpc) [reply bucket/cpu-z.json:43]: Needs new line @RavenMacDaddy by the looks of it

Comment (RavenMacDaddy) [reply bucket/cpu-z.json:43]: Looking at both commits in edit mode there seems to be an empty line at the end, does there need to be two?

**Pull request merged**